docs/data/routes/docs/nextjs/page-routing/jss-routes/en.md

```diff
@@ -18,7 +18,7 @@ Here's a high-level overview of the routing process.
 In the diagram above, you can see how the Next.js route is applied to Sitecore JSS.
 
 * The `[[...path]].tsx` Next.js route will catch any path and pass this information along to `getStaticProps` or `getServerSideProps` on the `context` object (as a tokenized array).
-* The page props factory uses the path information to construct a normalized Sitecore item path. It then makes a request to the JSS [Layout Service](/docs/fundamentals/services/layout-service) to fetch layout data for the item.
+* The *Page Props Factory* uses the path information to construct a normalized Sitecore item path. It then makes a request to the [Sitecore Layout Service REST API or Sitecore GraphQL "Edge" schema](/docs/nextjs/data-fetching/data-services#page-props) to fetch layout data for the item.
```

Comment:
Is this talking about a concept or the PagePropsFactory class? If the latter, then we need to match class name in docs (ie "PagePropsFactory") and also it would be helpful to link to it in GitHub

Reply:
This is referring to the concept / label in the diagram right above. I used similar treatment on other diagram / label callouts (EE architecture I think), so this is just making that consistent.